lambdas/service/Makefile

```diff
@@ -9,12 +9,14 @@ config: .chalice/config.json
 .PHONY: wheels
 wheels: check_env
 	rm -rf vendor/jsonobject*
-	unzip $(azul_home)/lambdas/.wheels/jsonobject-0.9.9-cp36-cp36m-linux_x86_64.whl -d vendor
+	unzip $(azul_home)/lambdas/.wheels/jsonobject-0.9.9-cp38-cp38-linux_x86_64.whl -d vendor
 
 .PHONY: deploy
 # TODO: Rename target: https://github.com/DataBiosphere/azul/issues/1654
 deploy: check_branch check_python check_aws config wheels
 	python -m azul.changelog vendor
+	# Ensure all .pyc files are present for a consistent deployment package
```

Comment:
Same here

Reply:
My mistake... I had misread the diff. I didn't realize that you had suggested a change. Now that I see that, your response makes a lot more sense.